plotter/bubbles_new_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter
+
+import (
+	"image/color"
+	"log"
+	"math/rand"
+	"os"
+	"testing"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/internal/cmpimg"
+	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
+	"gonum.org/v1/plot/vg/vgimg"
+)
+
+// ExampleScatter_bubbles draws some scatter points.
+// Each point is plotted with a different radius size depending on
+// some external criteria.
+func ExampleScatter_bubbles() {
+	rnd := rand.New(rand.NewSource(1))
+
+	// randomPoints returns some random x, y points
+	// with some interesting kind of trend.
+	randomPoints := func(n int) XYs {
```

Comment:
This function can be replaced with the one currently in `ExampleBubbles`, which returns `XYZs` instead of `XYs`.

Reply:
Then `sc, err := NewScatter(scatterData)` will cause change in `Scatter` type.

Reply:
Can we add an `XY()` method to `XYZs` so that it implements the `XYer` interface in addition to the `XYZer` interface?